babel/src/test/resources/sql/big-query.iq

```diff
@@ -3657,4 +3915,5 @@ FROM items;
 
 !ok
 
+
```

Comment:
Did you mean to add this blank line back?

Reply:
They popped up with the auto formatting, I'll take it out thanks for pointing out.
Do you know if the IS_NAN ones are valid (taking out the extra space) else I'll remove those too.

Reply:
Those are valid, it should remove blank space at the end of lines. I introduced those. Oops.